src/esp/assets/MeshData.h

```diff
@@ -7,23 +7,70 @@
 #include <vector>
 
 #include "esp/core/esp.h"
+#include <Magnum/GL/Mesh.h>
+#include <Corrade/Containers/Optional.h>
+#include <Corrade/Containers/Reference.h>
+#include <Magnum/Trade/MeshData3D.h>
+#include <Magnum/Magnum.h>
+#include <Magnum/Math/Color.h>
+#include <Magnum/Mesh.h>
+#include "MeshData.h"
 
 namespace esp {
 namespace assets {
 
 //! Raw mesh data storage
 struct MeshData {
   //! Vertex positions
-  std::vector<vec3f> vbo;
+  std::vector<vec3f>    vbo;
   //! Vertex normals
-  std::vector<vec3f> nbo;
+  std::vector<vec3f>    nbo;
   //! Texture coordinates
-  std::vector<vec2f> tbo;
+  std::vector<vec2f>    tbo;
   //! Vertex colors
-  std::vector<vec3f> cbo;
+  std::vector<vec3f>    cbo;
   //! Index buffer
   std::vector<uint32_t> ibo;
 };
 
+//! Reference to vertices and
+//! Usage: (1) for creating collision mesh in Bullet
+struct CollisionMeshData {
```

Comment:
If it is a different kind of mesh data, can you please create a pair of new files (.h/.cpp)?

CollisionMeshData.h / .cpp

Reply:
Good point.